plugins/legacy-plugin-chart-partition/test/optionDescirption.test.jsx

```diff
@@ -0,0 +1,48 @@
+/**
```

Comment:
This is a duplicate.

![image](https://user-images.githubusercontent.com/1659771/81732450-5badea00-9445-11ea-8768-dab0508b1b85.png)